components/gitpod-protocol/src/gitpod-service.ts

```diff
@@ -473,6 +476,22 @@ export namespace GitpodServer {
         type: GitpodTokenType;
         scopes?: string[];
     }
+    export interface OnboardingState {
+        /**
+         * Whether this Gitpod instance is already confingured with SSO.
+         */
+        readonly isCompleted: boolean;
+
+        /**
+         * Whether any org is .
+         */
+        readonly hasAnyOrg: boolean;
+
+        /**
+         * Whether this
+         */
+        readonly hasAnyOrgWithInactiveSSO: boolean;
```

Comment:
Should this be `hasAnyOrgWithActiveSSO` instead?

Reply:
> Should this be hasAnyOrgWithActiveSSO instead?

Wouldn't that be equal to `isCompleted`? 

I recall, you'd be interested in the state of having any Org without verified SSO.

Reply:
Good point, I suppose I'm not sure how we'd use this then, we can drop for now and add later if we have a reason.

Reply:
Thought about it, it seems to be redundant, or at least less helpful information. 

If you know that the onboarding is incomplete, and there is at least one org, you'd try to fetch the org, and eventually the SSO configs by org ID.

Reply:
Dropped it.